include/zenoh-pico/private/system.h

```diff
@@ -15,7 +15,7 @@
 #ifndef _ZENOH_PICO_SYSTEM_H
 #define _ZENOH_PICO_SYSTEM_H
 
-#if (ZENOH_LINUX == 1) || (ZENOH_MACOS == 1)
+#if (ZENOH_LINUX == 1) || (ZENOH_MACOS == 1) || (ESP32 == 1) || (ARDUINO_ARCH_ESP32 == 1)
```

Comment:
It would be preferable to use the same naming convention:
ESP32 -> ZENOH_ESP32
ARDUINO_ARCH_ESP32 -> ZENOH_ARDUINO_ESP32